developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -0,0 +1,732 @@
+---
+title: "Use Aptos CLI"
+id: "use-aptos-cli"
+---
+
+# Use Aptos CLI
+
+The `aptos` tool is a command line interface (CLI) for debugging, development, and node operation.
+This document describes how to use the `aptos` CLI tool. To install the CLI, see [Install Aptos CLI](install-aptos-cli).
+
+### Command Line Help
+Command line help is available.  Type `aptos help` or `aptos --help` to see the available command options.
+```bash
+$ aptos help
+aptos 0.2.0
+Aptos Labs <opensource@aptoslabs.com>
+CLI tool for interacting with the Aptos blockchain and nodes
+
```

Comment:
This looks verbatim.  I want to spend some time updating the docs after this is pushed to reflect some updates

Reply:
Yes. https://linear.app/aptoslabs/issue/DOC-45/update-aptos-cli-doc is our reference ticket.